src/graphql/definitions/website/section.graphql

```diff
@@ -59,6 +59,7 @@ input WebsiteSectionsQueryInput {
   status: ReferenceStatus = active
   parentId: Int
   siteId: String
+  ids: [Int]
```

Comment:
The `parentId` input seems to contradict the `ids` input. For example, a `parentId` value could be passed along with `ids` that aren't children, returning unexpected results. I would split this into a separate query and input for now... unless/until we create a generic query input